.github/workflows/container-image.yml

```diff
@@ -70,13 +70,23 @@ jobs:
     - uses: actions/checkout@v3
     - name: Build the client image
       run: make build-client
+    - name: Upload client image
+      uses: ishworkh/docker-image-artifact-upload@v1
+      with:
+        image: "quay.io/samba.org/samba-client:latest"
+        retention_days: 1
```

Comment:
Could you explain this a bit - where does it upload to? If it actually uploads to quay.io under the latest tag we're possibly promoting the image to a public repo prematurely.  If it's a local cache of sorts, I would also appreciate a comment here explaining as much.

Reply:
Upload/Download happens locally with GitHub artifacts. Added necessary comments to explain the process.